components/HeroUnit.js

```diff
@@ -84,3 +86,67 @@ function HeroUnitFooter () {
     </div>
   )
 }
+
+class HeroUnitBg extends React.Component {
+  componentDidMount () {
+    const { delay } = this.props
+    const bgHeroSet = ReactDOM.findDOMNode(this)
+    const img = bgHeroSet.querySelectorAll('img')
+
+    let pointer = 0
+    const maxItems = img.length
+
+    setInterval(() => {
```

Comment:
When using `setInterval` in `componentDidMount`, it’s best to save a reference to the interval ID, and do a `clearInterval` in `componentWillUnmount`.